src/selectors/icons.js

```diff
@@ -3,18 +3,18 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
-import { createSelector } from 'reselect';
 import type {
-  IconWithClassName,
   IconState,
   Selector,
   DangerousSelectorWithArguments,
 } from 'firefox-profiler/types';
 
 /**
  * A simple selector into the icon state.
+ * It returns a map that matches icon to the icon class name.
  */
-export const getIcons: Selector<IconState> = (state) => state.icons;
+export const getIconsWithClassNames: Selector<IconState> = (state) =>
```

Comment:
It would be better to use IconsWithClassNames instead of IconState, it's more meaningful.